packages/template/src/lib/stack-app/apps/implementations/server-app-impl.ts

```
@@ -967,6 +994,34 @@ export class _StackServerAppImplIncomplete<HasTokenStore extends boolean, Projec
+      const result = Result.orThrow(await this._serverTeamItemsCache.getOrWait([options.teamId, options.itemId], "write-only"));
+      return this._serverItemFromCrud({ type: "team", id: options.teamId }, result);
+    } else {
+      const result = Result.orThrow(await this._serverCustomItemsCache.getOrWait([options.customId, options.itemId], "write-only"));
+      return this._serverItemFromCrud({ type: "custom", id: options.customId }, result);
+    }
+  }
+
+  // IF_PLATFORM react-like
+  useItem(options: { itemId: string, userId: string } | { itemId: string, teamId: string } | { itemId: string, customId: string }): ServerItem {
+    if ("userId" in options) {
+      const result = useAsyncCache(this._serverUserItemsCache, [options.userId, options.itemId] as const, "app.useItem(user)");
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)